src/FffGcodeWriter.cpp

```diff
@@ -520,6 +522,7 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
         for (unsigned int each_extruder_need_prime : extruder_order)
         {
             setExtruder_addPrime(storage, gcode_layer, layer_nr, each_extruder_need_prime);
+            last_extruder_on_initial_raft_layer = each_extruder_need_prime;
```

Comment:
I will use `each_to_be_primed_extruder_nr`, the word `current` occurs too often...